.github/.kubescape/exceptions.json

```diff
@@ -79,6 +79,131 @@
         "controlID": "C-0036"
       }
     ]
+  },
+  {
```

Comment:
can you please add a short explanation for these exceptions so we don't loose track of why they were added?